kivy/effects/kinetic.py

```diff
@@ -177,6 +189,6 @@ def update_velocity(self, dt):
             self.velocity = 0
             return
 
-        self.velocity -= self.velocity * self.friction
+        self.velocity -= self.velocity * self.friction * dt / self.std_dt
```

Comment:
How is this different than simply setting `friction` to `0.05/0.017`?

Reply:
The difference is the consideration of the parameter dt. 
This value describes the elapsed time and is not constant. 
See the values for dt above.

Therefore, friction must always be applied over time and not as a constant. The value self.std_dt here only ensures that the previous value for friction can still be used.

Reply:
seems sensible to me, correcting for the actual dt vs expected dt should give a more consistent experience accross devices.

Reply:
Ok. What confused me is that when searching I only saw `friction` used with `dt`. So it wasn't clear to me why you couldn't replace `self.friction * dt / self.std_dt` with `self.friction * dt` but simply adjust `friction` to account for the variance on each device.

But I guess you're saying that conceptually they have different meanings so conceptually we should be able to set both `friction` and `std_dt` independently, even if in practice they are only ever used together as one factor? If I understood this correctly, it should be ok.